config/index.js

```diff
@@ -25,18 +33,33 @@ function printConfig (config) {
 *    ]
 *
 */
-module.exports = function (config, overrides) {
-  config.load = function () {
+module.exports = function (config, configDirPath, overrides) {
```

Comment:
We might as well just call `load` directly after we perform the `register`. I think some of the others have been bitten by not calling `load` after requiring the config in the past so performing it automatically will be a welcome change.